marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-admin-role.json

```diff
@@ -1,5 +1,5 @@
 {
-  "role-name": "%%mlDataHubAdminRole%%",
+  "role-name": "data-hub-admin-role",
   "description": "A role can deploy modules to a data hub instance.",
```

Comment:
Can you change this description as noted in the JIRA ticket too? This current description isn't really correct to start with, and I think we now want it to note that the role is deprecated and shouldn't be used.